extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1226,6 +1226,8 @@ public static function getCurrentStagingHost( $dbName='', $default='', $host = n
 	static public function getLocalEnvURL( $url, $forcedEnv = null ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 8, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)